src/08_comprehensions.py

```diff
@@ -36,6 +45,6 @@
 x = input("Enter comma-separated numbers: ").split(',')
 
 # What do you need between the square brackets to make it work?
-y = []
+y = [int(a) for b in x for a in b if int(a)%2 == 0]
```

Comment:
Try this patter for this whole file. This is a "List comprehension" and a really easy way to build and manipulate lists.